bundle.Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM scratch
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
-LABEL operators.operatorframework.io.bundle.package.v1=kogito-operator
+LABEL operators.operatorframework.io.bundle.package.v1=rhpam-kogito-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha,1.x
```

Comment:
What OLM channels we should have for the product?

Reply:
That's a good question and well spotted. We should have the 7.x only for now, IMO. As soon as we need more, we can add them in the future. I'd avoid too many.

Reply:
set channel to 7.x